speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionMaker.inl

```diff
@@ -50,6 +53,19 @@ void DecisionMaker<SocketBlackBox, SDAConfig>::Initialize(tCarElt* p_initialCar,
 
     BlackBoxData initialData(p_initialCar, p_initialSituation, 0, nullptr, 0);
     BlackBox.Initialize(initialData, p_testSituations, p_testAmount);
+
+    m_fileBufferStorage = new FileDataStorage(Config.GetDataCollectionSetting());
```

Comment:
Does this really need to be heap allocated? We can just reset the bufferstorage after a run right?